packages/dds/cell/package.json

```diff
@@ -92,6 +92,10 @@
   },
   "typeValidation": {
     "version": "2.0.0",
-    "broken": {}
+    "broken": {
+      "ClassDeclaration_SharedCell": {
```

Comment:
It's not okay to break compat here. All your API changes should be possible without breaking by making rollback optional, you're not using the return value of delete anyway, and localOpMetadata can still be unknown.